src/apps/api/views/competitions.py

```diff
@@ -201,6 +209,22 @@ def email_all_participants(self, request, pk):
         batch_send_email.apply_async((comp.pk, content))
         return Response({}, status=status.HTTP_200_OK)
 
+    @action(detail=True, methods=['GET',],)
+    def get_csv(self, request, pk,):
+        comp = Competition.objects.get(pk=pk)
```

Comment:
you can use `self.get_object()` here and it should leverage `self.get_queryset()` as well

also, do we need to do any permissions checks or anything before letting folks download this? Maybe they should be a creator/admin of the competition?

Reply:
It seems like it's already limited to be people who are logged in. Do we know who we are thinking will be using this data? Admins seem like a good place to start. Do we think it will be useful for people are competing in the competition to have access to this as well?

Reply:
We may want competitors to be able to download the scores, or anonymized scores, or some other modes later. I think admins is also a pretty good place to start, I am not certain if we have a nice helper function for this already or not.. may be an OK time to create a helper/utility function to check `Competition.check_user_admin(user)`